lib/feature/home_page/widgets/pokemon_tile.dart

```diff
@@ -22,25 +23,30 @@ class _PokemonTileState extends State<PokemonTile> {
   PokemonType? thisTileTypes;
   @override
   void initState() {
-    http.get(Uri.tryParse('${widget.thisPokemon.url}') ?? Uri()).then((value) {
-      if (value.statusCode == 200) {
-        var result = jsonDecode(value.body);
-        List SubTypeMap = result['types'];
-        setState(() {
-          thisTileTypes = PokemonType(
-            id: result['id'],
-            subTypes: SubTypeMap.map((e) => SubType(
-                slot: result['slot'],
-                type: SpecificType(
-                  name: e['type']['name'],
-                  url: e['type']['url'],
-                ))).toList(),
-          );
+    if (mounted) {
```

Comment:
put the mounted check inside the post frame callback. Reason is that if this holds true, and then, for some reason, the widget suddenly has been unmounted after the mounted check, everything inside the callback will still be called.

Reply:
![Screenshot 2022-07-27 at 1 38 08 PM](https://user-images.githubusercontent.com/47196695/181169569-b037d9e3-8d17-48d9-bb5a-37e33e5aacf3.png)
for example. Still encountered the error 😛